lib/cookieContentHandler.js

```diff
@@ -133,6 +139,10 @@ CookieContentHandler.prototype.add = function(req, res, tupleToAdd) {
     options.domain = this.domain;
   }
 
+  if (this.cookieSameSite && !sameSiteNotAllowed(req.get('User-Agent'))) {
+    options.sameSite = 'none';
```

Comment:
Secure as well? We need to add secure to fully comply with Chrome changes.

Reply:
added